AlwaysOnKotlin/Wearable/build.gradle

```diff
@@ -20,13 +20,13 @@ plugins {
 }
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
There is no Wear 31. :)

Reply:
This is a curious point, I was going to ask about this. `lint`/Android Studio complains loudly if not targeting and compiling against the latest version, even though there currently isn't a Wear API 31. My guess would be that compiling and targeting against the latest API should be fine?

Reply:
Hmm... that an interesting question... I'm not 100% sure there isn't something that could happen. Let's verify.

Reply:
Updated in the latest set of commits, with the context of https://issuetracker.google.com/issues/196420849